lighthouse-cli/package.json

```diff
@@ -8,6 +8,8 @@
     "typescript": "^2.0.3"
   },
   "dependencies": {
-    "@types/node": "^6.0.45"
+    "@types/node": "^6.0.45",
+    "express": "^4.14.0",
```

Comment:
IMO express is very minimalist already. I think the trade off is writing less code or less dependencies. Express lets us do simple routing and deals with basic security issues in single lines. (current code can be cut down further).